### ISSUE TITLE
[Enhancement] Include validation receipts in gossip, make them queryable

### ISSUE THREAD

**Issue by mattyg** [holochain/holochain#4175]
Make validation receipts available to be queried and include them in gossip, so they can be exposed to others who want to see the social perspective on validity of actions.

**Comment by 8e8b2c** [comment]
I have an upcoming client who needs strong security guarantees, and will want as much transparency on the health of their network and the authenticity of the data within it, as is feasibly possible. Being able to display validation receipts for any particular action will get us part of the way there, because this informs:
- Who we are trusting to assess the authenticity of a given action
- How many peers have added their weight to the validation

Both of these parameters are potentially useful in evaluating the risk of collusion or a sybil attack.

**Comment by mattyg** [comment]
Yeah this definitely needs more thought. I guess in general you wouldn't care about other peers' validation receipts because you're running validation yourself. In a "dht explorer" type app however you might care about other peers' validation receipts because you're not actually participating in the network, just an outside observer -- but maybe this is a wrong approach
